packages/playback-core/src/index.ts

```diff
@@ -438,6 +435,27 @@ export const loadMedia = (
   }
 };
 
+function seekInSeekableRange(event: Event) {
+  const mediaEl = event.target as HTMLMediaElement;
+  const startTime = muxMediaState.get(mediaEl)?.startTime;
+  if (!startTime) return;
+
+  if (inSeekableRange(mediaEl.seekable, mediaEl.duration, startTime)) {
+    // Setting preload to `none` from `auto` was required on iOS to fix a bug
+    // that caused no `timeupdate` events to fire after seeking ¯\_(ツ)_/¯
+    const wasAuto = mediaEl.preload === 'auto';
+    if (wasAuto) {
```

Comment:
question(blocking): Just confirming, this issue does _**not**_ happen when `preload = 'metadata'`, correct?

Reply:
yes, double checked that. it's working correctly with `preload=metadata`